help/destinations/catalog/crm/salesforce.md

```diff
@@ -137,41 +150,58 @@ When you are finished providing details for your destination connection, select
 
-    ![Add new mapping](../../assets/catalog/crm/salesforce/add-new-mapping.png)
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Platform UI screenshot example for Add new mapping.](../../assets/catalog/crm/salesforce/add-new-mapping.png)
 
-1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select attributes]** category and add the mappings desired.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select identity namespace]** category and select `crmID`.
+![Platform UI screenshot example for Source mapping.](../../assets/catalog/crm/salesforce/source-mapping.png)
 
-    ![Source mapping](../../assets/catalog/crm/salesforce/source-mapping.png)
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** category and select `SalesforceId`.
+    * **[!UICONTROL Select identity namespace]**: select this option to map your source field to an identity namespace from the list.
+    ![Platform UI screenshot showing Target mapping for SalesforceId.](../../assets/catalog/crm/salesforce/target-mapping-salesforceid.png)
+    * Add the following mapping between your XDM profile schema and your [!DNL Salesforce CRM] instance:
 
-1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+    |XDM Profile Schema|[!DNL Salesforce CRM] Instance| Mandatory |
+    |---|---|---|
+    |`crmID`|`SalesforceId`|Yes|
 
-    ![Target mapping using SalesforceId](../../assets/catalog/crm/salesforce/target-mapping-salesforceid.png)
+    * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you have defined in the **[!UICONTROL Attribute name]** field. Refer to [[!DNL Salesforce CRM] documentation](https://help.salesforce.com/s/articleView?id=sf.custom_field_attributes.htm&type=5) for guidance on supported attributes.
+    ![Platform UI screenshot showing Target mapping for LastName.](../../assets/catalog/crm/salesforce/target-mapping-lastname.png)
 
-1. For custom attributes, in the select target field window, select the target field and choose the **[!UICONTROL Select custom attributes]** category, Next provide the desired target attribute name and add the mappings desired.
+    * If you are working with *Contacts* within your segment, refer to the Object Reference in Salesforce for [Contact](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_contact.htm) to define mappings for the fields to be updated.
+    * Mandatory fields can be identified by searching for the word *Required* which is mentioned in its field description.
```

Comment:
Suggestion to rephrase: 

"You can identify mandatory fields by searching for the word *Required*, which is mentioned in field descriptions in the link above."